### PR TITLE
Fix #40

### LINKED ISSUE #40: update name of the polish filters
https://github.com/MajkiIT/polish-ads-filter#about
2018-04-15-nano panel sterowania

### 1.
```
https://raw.githubusercontent.com/MajkiIT/polish-ads-filter/master/polish-adblock-filters/adblock.txt
```
from
```
Adblock & uBlock polish filter
```
to
```
Official Polish filters for AdBlock, uBlock Origin & AdGuard

Oficjalne polskie filtry do AdBlocka, uBlocka Origin i AdGuarda
```

##

### 2.
```
https://raw.githubusercontent.com/MajkiIT/polish-ads-filter/master/cookies_filters/adblock_cookies.txt
```
from 
```
Adblock & uBlock polish cookies filter
```
to
```
Polskie Filtry Ciasteczkowe

Polish Cookies Filters
```

##

### 3.
```
https://raw.githubusercontent.com/PolishFiltersTeam/PolishAnnoyanceFilters/master/PPB.txt
```
from
```
Polish Annoyance Filters
```
to
```
Polskie Filtry Elementów Irytujących

Polish Annoyance Filters
```

##

### 4.
```
https://raw.githubusercontent.com/MajkiIT/polish-ads-filter/master/adblock_social_filters/adblock_social_list.txt
```
from
```
Polski social filtr
```
to
```
Polskie Filtry Społecznościowe

Polish Social Filters
```

##

### 5.
```
https://raw.githubusercontent.com/PolishFiltersTeam/PolishRSSFilters/master/polish_rss_filters.txt
```
from
```
Polish RSS Filters
```
to
```
Polskie Filtry RSS

Polish RSS Filters
```

##

### 6.
```
https://raw.githubusercontent.com/MajkiIT/polish-ads-filter/master/polish-adblock-filters/adblock_adguard.txt
```
from
```
Adblock & uBlock polish filter - AdGuard supplement
```
to
```
Official Polish filters for AdBlock, uBlock Origin & AdGuard - supplement for AdGuard

Oficjalne polskie filtry do AdBlocka, uBlocka Origin i AdGuarda - uzupełnienie do AdGuarda
```

---

update homepage address for `Official Polish filters for AdBlock, uBlock Origin & AdGuard`, `Polskie Filtry Ciasteczkowe` and `Polskie Filtry Społecznościowe` to

```
https://github.com/MajkiIT/polish-ads-filter#polish-filters-for-adblock-ublock-origin--adguard
```

### PR DESCRIPTION
**Pull request by AdamWr**
#40

### CODE REVIEW

File: filters/ThirdParty/filter_216_PolskiFiltr/metadata.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "filterId": 216,
-  "name": "Adblock & uBlock polish filter",
+  "name": "Official Polish filters for AdBlock, uBlock Origin & AdGuard",
   "description": "polski filtr",
```
Comment (ameshkov) [inline filters/ThirdParty/filter_216_PolskiFiltr/metadata.json]: Do we need to change the description?

Comment (AdamWr) [reply filters/ThirdParty/filter_216_PolskiFiltr/metadata.json]: I'm not sure, but other filter lists have similar description, for example:
https://github.com/AdguardTeam/FiltersRegistry/blob/master/filters/ThirdParty/filter_203_hufilter/metadata.json#L4
and
https://github.com/AdguardTeam/FiltersRegistry/blob/master/filters/ThirdParty/filter_201_CzechList/metadata.json#L4

Comment (ameshkov) [reply filters/ThirdParty/filter_216_PolskiFiltr/metadata.json]: We usually just use the description from the filter file

Comment (AdamWr) [reply filters/ThirdParty/filter_216_PolskiFiltr/metadata.json]: So what should I add here?
This? - `Oficjalne polskie filtry do AdBlocka, uBlocka Origin i AdGuarda`
https://github.com/MajkiIT/polish-ads-filter/blob/master/polish-adblock-filters/adblock.txt#L3

Comment (ameshkov) [reply filters/ThirdParty/filter_216_PolskiFiltr/metadata.json]: Yeah, that's right

Comment (AdamWr) [reply filters/ThirdParty/filter_216_PolskiFiltr/metadata.json]: Ok, I changed description and homepage addresses which was mentioned here https://github.com/AdguardTeam/FiltersRegistry/issues/40#issuecomment-381832874 and here https://github.com/AdguardTeam/FiltersRegistry/issues/40#issuecomment-383333269

**Pull request merged**